backend/src/apiserver/template/template_test.go

```diff
@@ -172,6 +173,8 @@ func TestToSwfCRDResourceGeneratedName_EmptyName(t *testing.T) {
 }
 
 func TestScheduledWorkflow(t *testing.T) {
+	proxy.InitializeConfigWithEnv()
```

Comment:
In tests, it'd probably be safer to initialize with an empty proxy config so that it's deterministic (not dependent on if you have those environment variables set locally).

Reply:
Yeah, `proxy.EmptyConfig()` so that the user's environment variables don't impact the tests.